### DISCUSSION TITLE
Feature-Request: Elsa 2.0 - Trigger for Entity Changed

### DISCUSSION THREAD

**Discussion by Rafael94** [elsa-workflows/elsa-core#497] [Ideas]
Hello,

in my application I use entities very often, for example ticket, customer, devices etc.  These are added, changed or deleted.  I think it will be the same with other projects.

I would therefore welcome if a trigger is provided directly in Elsa, so that not every developer has to create one for himself.

## Example: ##
Name: EntityChangedTrigger
- EntityName (Fullname of the Class?)
- Action (Added, Updated, Deleted)

In the best case the developer could specify the supported entities and the user gets these entities displayed in the dashboard as a selection. If this would set the context (WithContextType<TEntity>) automatically, it would be perfect.

Translated with www.DeepL.com/Translator

**Comment by sfmskywalker** [comment]
I like it. In fact, I will need something like this for a project I'm working on myself sometime in the future. I'm not yet sure if it would belong to the core distribution of Elsa, or as a separate repository + package, but we'll see. Thanks for proposing the feature!

**Reply by Rafael94** [reply]
In my opinion the trigger could go into Core. It has no dependencies to external libraries. Later on you could still outsource it, if it becomes more.
